pages/04.knowledgebase/01.installation/how-to-install-mautic-on-a-localhost/how-to-install-mautic-using-ddev/item.md

```diff
@@ -136,4 +187,8 @@ This will give you a lot of information about your containers, including the URL
 
 [ddev]: (https://github.com/drud/ddev)
 [install-docker]: (https://ddev.readthedocs.io/en/stable/users/docker_installation/)
-[install-ddev]: (https://ddev.readthedocs.io/en/stable/#installation)
\ No newline at end of file
+[install-ddev]: (https://ddev.readthedocs.io/en/stable/#installation)
+[tz-values]: (https://en.wikipedia.org/wiki/List_of_tz_database_time_zones)
+[composer]: (https://getcomposer.org)
+[cli-commands]: (https://docs.mautic.org/en/troubleshooting/command-line-tools-cli)
+[mkcert]: (https://github.com/FiloSottile/mkcert)
```

Comment:
```suggestion
[mkcert]: <https://github.com/FiloSottile/mkcert>
```

Reply:
Hey @RCheesley I can't approve this one since it's marked as "Outdated" for some reason? :confused:

Reply:
That means that the code in the PR has changed since the comment was left :) Will take a look!